history/thmutil.child.controls.md

```diff
@@ -0,0 +1,6 @@
+* @barnson: Add child control support to ThmUtil.
+  * Add Panel control to hold children.
+  * Billboards can now host panels for mixing text and graphics.
+  * Graphic buttons can now have localized text drawn on them.
+  * Graphic buttons now support four states: unselected, hover, selected, focused.
+  * Controls now have a DisableAutomaticBehavior attribute to turn off variable getting and setting, EnableCondition, VisibleCondition, and conditional Text elements.
```

Comment:
This was incredibly difficult to review. There really should have been at least one separate commit for each bullet point here.  Case in point: I can't give inline comments because the diff on thmutil.cpp is too large for GitHub to show.